PyExpLabSys/drivers/scpi.py

```diff
@@ -18,23 +20,27 @@
 
 class SCPI(object):
     """ Driver for scpi communication """
-    def __init__(self, interface, device='', tcp_port=5025, hostname='', baudrate=9600,
-                 visa_string='', line_ending='\r'):
+    def __init__(self, interface, device='', tcp_port=5025, hostname='',
+                 baudrate=9600, visa_string='', gpib_address=None, line_ending='\r'):
         self.device = device
         self.line_ending = line_ending
         self.interface = interface
         if self.interface == 'file':
```

Comment:
Should this be an if-elif-elif-else: Error sort of construct ?

Reply:
I am not sure I understand this? The change is just to allow for a gpib address, that defaults to None because it is only used for one particular interface.

Reply:
Yeah, sorry. It was a comment on the general structure of the interface selection, which can be freely ignored.